src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -370,6 +446,11 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         UpdateSettings(_settingsSource);
     }
 
+    void MainPage::BreadcrumbBar_ItemClicked(Microsoft::UI::Xaml::Controls::BreadcrumbBar const& /*sender*/, Microsoft::UI::Xaml::Controls::BreadcrumbBarItemClickedEventArgs const& /*args*/)
+    {
+        _lastProfilesNavState.Profile().CurrentPage(L"Base");

```

Comment:
Since the only use case right now is moving back to `Profiles_Base` from somewhere the Advanced/Appearance pages, this "works" but we really should be using `args.Index()` and the current page to figure out where to navigate to